cmd/serverFlags.go

```diff
@@ -0,0 +1,17 @@
+package cmd
+
+import (
+	"github.com/puppetlabs/wash/cmd/internal/server"
+	"github.com/spf13/cobra"
+)
+
+// SetFlags adds server flags to the specified command.
+func serverFlags(cmd *cobra.Command) *server.Opts {
```

Comment:
I think this should be in `cmd/server.go`, and renamed to `addServerFlags`. We should also add a `newServer(cmd)`-type method in `server.go` that creates the server object from these flags. This way, we can still bind to Viper, which we will need to do because that is the correct way to parse them from the config file (see https://github.com/puppetlabs/wash/blob/master/config/core.go#L31-L38. Note that with the current version of Wash, these options can be set from the environment). 

I like the symmetry here in binding the option variables to the flag; however, the `makeRootMain`/`makeServerMain` code is weird (but also the only way I can think of to get these set in one place w/ the options, so I get why you added it).

Reply:
I'll experiment more. I'm not sure why the Viper bindings were such an issue when I tried to centralize them.

Reply:
Running into https://github.com/spf13/viper/issues/233.

Reply:
Fixed.